### PR TITLE
Order finish activity

### PR DESCRIPTION
**Pull request by nir-ya**
new activity and partial functionality of calling to make the order

### CODE REVIEW

File: app/src/main/java/com/example/myapplication/Constants.java [hunk 1]
```diff
@@ -8,10 +8,10 @@
     static final String ORDERS = "OpenOrders";
     static final String MANOT_SUBCOLLECTION = "Manot";
     static final String MONEY_MADE = "הכסף שנצבר: %s מתוך 70 שקלים";
-    static final String LOCKED_TEXT = "נעול";
+    static final String LOCKED_TEXT = "הזמן!";
```
Comment (liav25) [inline app/src/main/java/com/example/myapplication/Constants.java]: Should reformat the constane reference. Myabe "READY_TEXT"?

File: app/src/main/java/com/example/myapplication/Constants.java [hunk 2]
```diff
@@ -8,10 +8,10 @@
...
     static final String JOIN_TEXT = "הצטרף!";
     static final String READY_TEXT = "מוכן לשילוח";
-    static final String ORDER_OUT = "ההזמנה יצאה";
+    static final String ORDER_OUT = "שעת ההזמנה הגיעה. מישהו צריך לקחת יוזמה";
```
Comment (liav25) [inline app/src/main/java/com/example/myapplication/Constants.java]: Same here

File: app/src/main/java/com/example/myapplication/OrderFinishActivity.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_order_finish);
+    }
+
+    public void dialShevach(View view) {
+        Intent dialIntent = new Intent(Intent.ACTION_DIAL,
+                Uri.parse("tel:" + getString(R.string.shevach_phone_number)));
```
Comment (liav25) [inline app/src/main/java/com/example/myapplication/OrderFinishActivity.java]: Amazing

File: app/src/main/res/layout/activity_order_finish.xml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:background="@drawable/border_button"
+        android:fontFamily="@font/varela_round"
+        android:padding="10dp"
+        android:textStyle="bold"/>
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
```
Comment (liav25) [inline app/src/main/res/layout/activity_order_finish.xml]: Great job using conatraints

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -68,5 +68,15 @@
     <string name="invalid_password_msg">בחר סיסמה בעלת 6 תווים לפחות</string>
     <string name="invalid_email_msg">כתובת הe-mail שהכנסת לא תקינה</string>
     <string name="order_list_item_title">הזמנה לשעה: %s</string>
+    <string name="final_order_details">
+        פיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה\nפיתה
```
Comment (liav25) [inline app/src/main/res/values/strings.xml]: Guess this is for testing?

File: app/src/main/java/com/example/myapplication/Constants.java (hunk 1 above)
Comment (nir-ya) [reply app/src/main/java/com/example/myapplication/Constants.java]: You're right, I'll take care of it after merging.

File: app/src/main/java/com/example/myapplication/Constants.java (hunk 2 above)
Comment (nir-ya) [reply app/src/main/java/com/example/myapplication/Constants.java]: You're right, I'll take care of it after merging.

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (nir-ya) [reply app/src/main/res/values/strings.xml]: exactly

**Pull request merged**